tutorials/surrogate/cifar10_example.py

```diff
@@ -0,0 +1,429 @@
+#!/usr/bin/env python3
+
+import random
+import os
+import sys
+import enum
+from typing import Union, Dict, Tuple, Generator
+
+import torch
+from torch import nn
+from torch.utils.data import DataLoader, TensorDataset
+
+from torchvision.datasets import CIFAR10
+from torchvision.transforms import Compose, ToTensor, Normalize
+from torchmetrics import Accuracy
+
+from mpi4py import MPI
+
+from propulate import Islands
+from propulate.utils import get_default_propagator
+
+# from tutorials.surrogate.mock_surrogate import MockSurrogate
+# from tutorials.surrogate.log_surrogate import LogSurrogate
+from tutorials.surrogate.static_surrogate import StaticSurrogate
+# from tutorials.surrogate.dynamic_surrogate import DynamicSurrogate
+
+
+GPUS_PER_NODE: int = 1
+
+log_path = "torch_ckpts"
+
+sys.path.append(os.path.abspath('../../'))
+
+
+class Permutation(enum.Enum):
```

Comment:
Can you add a docstring to briefly explain what the role of `Permutation` is in this example?